doc/nrf/releases/release-notes-latest.rst

```diff
@@ -247,6 +247,35 @@ The following list summarizes the most important changes inherited from upstream
     * Fixed a discovery response formatting.
     * Updated a few API functions to accept a const pointer when appropriate.
 
+* Bluetooth:
+
+  * Bluetooth Host:
+
+    * Fixed a crash where an ATT timeout occurred on a disconnected ATT channel.
+    * Removed definitions and functions that were deprecated since the v2.3.0 release.
+    * Changed the pairing procedure to fail pairing when both sides have the same public key.
+    * Fixed an issue where GATT requests might deadlock RX thread.
+    * Fixed an issue where a fixed passkey that was previously set could not be cleared.
+    * Fixed an issue where callbacks for "security changed" and "pairing failed" were not always called.
+    * Changed the pairing procedure to fail early if the remote device cannot achieve the required security level.
+    * Fixed an incomplete bond overwrite during the pairing procedure when the peer is not using the IRK stored in the bond.
+    * Fixed an issue where GATT notifications and Writes Without Response might be sent out of order.
+    * Changed buffer ownership of :c:func:`bt_l2cap_chan_send`.
+      The application must now release the buffer for all returned errors.
+
+  * Bluetooth Mesh:
+
+    * Fixed Friendship counter error handling.
+    * Added replay check on segmented transport messages.
+    * Added poll callback for Friend callback.
+    * Added BabbleSim test suite.
+    * Storing health server publish period divisor persistently.
+    * Added GATT Proxy callbacks.
+    * Added address matching for Config Client response messages.
+    * Added basic model behavior in Mesh sample.
+    * Added composition data traversal API.
+    * Added acknowledged model message API.
+    * Fixed the Config Server's LPN time reporting.
```

Comment:
```suggestion
    * Fixed error handling for the Friendship counter.
    * Added a replay check on segmented transport messages.
    * Added a poll callback for the Friend callback.
    * Added BabbleSim test suite.
    * Updated the publish period divisor of the health server to be stored persistently.
    * Added GATT Proxy callbacks.
    * Added address matching for Config Client response messages.
    * Added basic model behavior in the Mesh sample.
    * Added a composition data traversal API.
    * Added an acknowledged model message API.
    * Fixed the Config Server's LPN time reporting.
```

Reply:
Did I resolve "health server publish period divisor" correctly? Five consecutive nouns is a bit much. 😉

Reply:
@trond-snekvik Does this look ok?

Reply:
Yep, still makes sense 👍